docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1442,7 +1514,7 @@ must restart the server for that change to take effect.
 |`fhirServer/bulkdata/jobParameters/cos.api.key`|Y|Y|
 |`fhirServer/bulkdata/jobParameters/cos.srvinst.id`|Y|Y|
 |`fhirServer/bulkdata/bulkDataBatchJobIdEncryptionKey`|Y|Y|
-|`fhirServer/bulkdata/isExportPublic`|Y|Y|
+|`fhirServer/bulkdata/isExportPublic`|N|Y|
```

Comment:
How is this not tenant specific?  I thought each tenant is able to override

Reply:
For now, it's not tenant specific, the config is always gotten from the default config. but we can change codes to make it tenant specific if need in the future.

Reply:
got it. It's where you call it there is no 'tenant' and therefore always default.